src/core/iomgr/pollset_windows.c

```diff
@@ -126,25 +125,25 @@ void grpc_pollset_reset(grpc_pollset *pollset) {
 }
 
 void grpc_pollset_work(grpc_exec_ctx *exec_ctx, grpc_pollset *pollset,
-                       grpc_pollset_worker *worker, gpr_timespec now,
+                       grpc_pollset_worker **worker_hdl, gpr_timespec now,
```

Comment:
So I'm a little confused - what's the point of the handle? It is dereferenced only to point it at a local variable and then back to NULL at the end of the function. It is never double-dereferenced, AFAICT.